tests/model_hub_tests/models_hub_common/constants.py

```diff
@@ -3,6 +3,13 @@
 # SPDX-License-Identifier: Apache-2.0
 
 import os
+import tempfile
+
+if os.environ.get('TFHUB_CACHE_DIR') is not None:
```

Comment:
Looks like it is just a dictionary, I may suggest to use "if 'TFHUB_CACHE_DIR' in os.environ"? As an alternate way if you want to use get, I suggest to use it like: tf_hub_cache_dir = os.environ.get('TFHUB_CACHE_DIR', os.path.join(...))